install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -92,6 +92,14 @@ instructions.
 . Add or expand the cluster. for example, if you want to add 3 masters with etcd
 colocated, scale up 3 master nodes.
 
+[IMPORTANT]
+====
+In the initial release of {product-title} version 3.11, the *_scaleup.yml_*
+playbook might not scale up etcd if you use CRI-O. This will be fixed in a
```

Comment:
The summary of that bug maybe a little confusing, the scale up etcd function is not working in all situations, not only `if you use CRI-O`